src/api/twitterApi.ts

```diff
@@ -0,0 +1,98 @@
+// see https://developer.twitter.com/en/docs/twitter-api/users/lookup/api-reference/get-users-by-username-username#tab1
```

Comment:
Comments at the top not necessary, we can google the package when looking for documentation